### PR TITLE
Remove superadmin feat

### LINKED ISSUE #737: Remove SuperAdmin functionality
- [ ] remove the logic that grants the permission in the auth script
- [ ] remove the admin_view perm for the 14 existing super admins

### PR DESCRIPTION
**Pull request by zashton**
Fixes #737

### CODE REVIEW

**Comment by CryptoGraffe** [comment]
yes!!

File: src/recoilState/app.ts [hunk 1]
```diff
@@ -210,30 +209,16 @@ export const rCircle = selectorFamily<ICircleState, number | undefined>({
...
 
       if (meOrPretend === undefined || circle === undefined) {
         console.error(
           'unable to load circle or current user - circle?',
           circle,
           'user?',
-          meOrPretend,
-          'impersonate?',
-          impersonate
+          meOrPretend
```
Comment (topocount) [inline src/recoilState/app.ts:221]: this naming :joy_cat:

**Review by topocount — DISMISSED** [review]
approved, pending e2e tests pass after this PR is rebased :wink:

**Comment by zashton** [comment]
> approved, pending e2e tests pass after this PR is rebased 😉

just rebased , tests passed 😄

**Review by topocount — COMMENTED** [review]
send it

**Pull request merged**